app/views/books/index.html.erb

```diff
@@ -9,6 +9,11 @@
       <%= book.title %> ,
       <%= t('simple_form.labels.author') %>
       <%= book.author %>
+      <%= link_to t('books.edit.link'), edit_book_path(book) %>
+      <%= link_to t('books.delete.link'),
+                  book,
+                  method: :delete,
+                  data: { confirm: t('books.delete.confirmation_text') } %>
```

Comment:
I would try to extract a helper for this so you could do `<%= delete_book_link(book) %>` that way you can practice using helper and the view will be a bit cleaner here